### ISSUE TITLE
Normalize docstring formatting

### ISSUE THREAD

**Issue by erikrose** [erikrose/blessings#76]
The format favored on master is…

```
"""Return a capability that sets the foreground color.

The capability is unparametrized until called and passed a number
(0-15), at which point it returns another string which represents a
specific color change. This second string can further be called to
color a piece of text and set everything back to normal afterward.

:arg num: The number, 0-15, of the color

"""
```

That's as PEP-257-compliant as practicable, deviating only when the summary line can't fit in 79 cols. Sphinx joins multi-line first lines uneventfully. Note the Sphinx-style `:arg` directives, the trailing blank line inside the quotes as per PEP 257, the triple quotes, and the summary line beginning on the same line as the opening quotes. Symbols referenced within the docstring should be double-tilde-quoted as the contents of the docstring should be ReST.

Blocks #72.

**Comment by jquast** [comment]
Mostly complete, only sequences.py remains and terminal.py is partially completed in branch blessed-integration-pep257-docstrings.

This is solved by the same branch as issue #84 (``blessed-integration-pep257-docstrings``), as reformating and rewriting the same docstrings would otherwise cause merge conflicts or further branch-on-branches due to pending PR's.

**Comment by jquast** [comment]
Closed by #100, review at http://1984.ws/blessings/index.html
